cmd/config.go

```diff
@@ -313,6 +315,48 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
 		return nil, err
 	}
 	servicesConfig.Kubelet.InfraContainerImage = infraPodImage
+
+	// Add the flexvolume mount to kublet ExtraBinds
+	enableFlexVolMount, err := getConfig(reader, "Enable the FlexVolume driver mount", "no")
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.ContainsAny(enableFlexVolMount, yesAnswer) {
+		// Allow the user to set custom flex volume mount points
+		FlexVolumeBind, err := getConfig(reader, "Flex volume bind mount path for the kubelet service", defaultFlexVolumeBindPath)
+
+		if err != nil {
+			return nil, err
+		}
+
+		// Set the --volume-plugin-dir extra_args for kubelet service
+		volPluginDir := strings.Split(FlexVolumeBind, ":")
+
+		extraArgs := map[string]string{"volume-plugin-dir": volPluginDir[1]}
+
+		servicesConfig.Kubelet.ExtraArgs = extraArgs
+
+		servicesConfig.Kubelet.ExtraBinds = append(servicesConfig.Kubelet.ExtraBinds, FlexVolumeBind)
+
+	}
+
+	// Add additional ExtraBinds
+	additionalBinds, err := getConfig(reader, "Add additional bind mounts for kubelet service", "no")
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.ContainsAny(additionalBinds, yesAnswer) {
```

Comment:
If you are going to use yesAnswer, can you please change it in the rest of the questions in the rke config